Add docs for OpenTelemetry Java (#5968)
getsentry/sentry-docs: src/platform-includes/performance/opentelemetry-setup/with-java-agent/exporter-messages/java.mdx

> In case you are using `sentry-opentelemetry-agent` without needing to use any OpenTelemetry exporters you can add
> the following environment variables to turn off exporters and stop seeing error messages about
> servers not being reachable in the logs.

Reviewer: ```suggestion
To use `sentry-opentelemetry-agent` without OpenTelemetry exporters, add
the following environment variables to turn off exporters and stop receiving error messages about
servers not being reachable in the logs.
```

Example log message:
```
ERROR io.opentelemetry.exporter.internal.grpc.OkHttpGrpcExporter - Failed to export spans. The request could not be executed. Full error message: Failed to connect to localhost/[0:0:0:0:0:0:0:1]:4317
ERROR io.opentelemetry.exporter.internal.grpc.OkHttpGrpcExporter - Failed to export metrics. The request could not be executed. Full error message: Failed to connect to localhost/[0:0:0:0:0:0:0:1]:4317
```

#### Traces

> To turn off exporting of traces you can set `OTEL_TRACES_EXPORTER=none` as environment variable
> see [OpenTelemetry GitHub](https://github.com/open-telemetry/opentelemetry-java/tree/main/sdk-extensions/autoconfigure#otlp-exporter-span-metric-and-log-exporters).

Reviewer: Please double check to make sure my updates haven't changed the meaning.
```suggestion
To turn off traces exporting, set `OTEL_TRACES_EXPORTER=none` as an environment variable.
See the [OpenTelemetry GitHub](https://github.com/open-telemetry/opentelemetry-java/tree/main/sdk-extensions/autoconfigure#otlp-exporter-span-metric-and-log-exporters).
```

#### Metrics

> To turn off exporting of metrics you can set `OTEL_METRICS_EXPORTER=none` as environment variable
> see [OpenTelemetry GitHub](https://github.com/open-telemetry/opentelemetry-java/tree/main/sdk-extensions/autoconfigure#otlp-exporter-span-metric-and-log-exporters).

Reviewer: ```suggestion
To turn off metrics exporting, set `OTEL_METRICS_EXPORTER=none` as an environment variable.
See the [OpenTelemetry GitHub](https://github.com/open-telemetry/opentelemetry-java/tree/main/sdk-extensions/autoconfigure#otlp-exporter-span-metric-and-log-exporters).
```
